src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509SubjectKeyIdentifierExtension.cs

```diff
@@ -122,7 +122,7 @@ private static byte[] GenerateSubjectKeyIdentifierFromPublicKey(PublicKey key, X
                     return X509Pal.Instance.ComputeCapiSha1OfPublicKey(key);
 
                 default:
-                    throw new ArgumentException(nameof(algorithm));
+                    throw new ArgumentException(SR.Arg_InvalidKeyIdentifierHashAlgorithm, nameof(algorithm));
```

Comment:
Based on prior art, (e.g. https://github.com/dotnet/corefx/blob/1bd332f2e5f559c2eab276acb0d9e4256085ec2f/src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Store.cs#L56), I think that this should be
```C#
throw new ArgumentException(SR.Format(SR.Arg_EnumIllegalVal, algorithm), nameof(algorithm));
```

And no new resource string required.